app/inpututils.cpp

```diff
@@ -270,6 +270,40 @@ bool InputUtils::cpDir( const QString &srcPath, const QString &dstPath )
   return true;
 }
 
+QString InputUtils::renameFile( const QString &srcPath, const QString &prefix )
+{
+  QFile file( srcPath );
+
+  if ( file.exists() )
+  {
+    QFileInfo info( srcPath );
+    QString newFilename;
+    QString currTime = QDateTime::currentDateTime().toUTC().toString( "yyyy-MM-dd_hh_mm_ss.zzz" );
+    if ( !prefix.isEmpty() )
+    {
+      newFilename = QString( "%1_%2.%3" ).arg( prefix ).arg( currTime ).arg( info.suffix() );
+    }
+    else
+    {
+      newFilename = QString( "%1.%2" ).arg( currTime ).arg( info.suffix() );;
+    }
+    QString newPath( info.absolutePath() + "/" + newFilename );
+
+    if ( file.rename( newPath ) )
```

Comment:
Also a nitpick - this could be also done as a static method `QFile::rename(oldpath, newpath)`